web/modules/custom/hoeringsportal_base_fixtures/src/Fixture/PublicMeetingFixture.php

```diff
@@ -145,17 +152,209 @@ public function load() {
+          $offset - 1))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'spots' => 87,
+      ],
+      range(1, (int) (new \DateTimeImmutable('this month'))->format('t')),
+    ));
+    $node->save();
+
+    $node = $node->createDuplicate();
+    $node->setTitle('Public meeting with orders');
+    $node->set('field_pretix_dates', [
+      [
+        'location' => 'The location',
+        'address' => 'Hack Kampmanns Plads 2, 8000 Aarhus C',
+        'registration_deadline_value' => (new \DateTimeImmutable('2024-12-31T00:00:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'time_from_value' => (new \DateTimeImmutable('2025-01-01T19:00:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'time_to_value' => (new \DateTimeImmutable('2025-01-01T21:00:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'spots' => 10,
+      ],
+      [
+        'location' => 'Another location',
+        'address' => 'Rådhuspladsen 1, 8000 Aarhus C',
+        'registration_deadline_value' => (new \DateTimeImmutable('2025-11-30T00:00:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'time_from_value' => (new \DateTimeImmutable('2025-12-01T15:00:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'time_to_value' => (new \DateTimeImmutable('2025-12-01T16:30:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'spots' => 10,
+      ],
+      [
+        'location' => 'The location',
+        'address' => 'Hack Kampmanns Plads 2, 8000 Aarhus C',
+        'registration_deadline_value' => (new \DateTimeImmutable('2025-11-30T00:00:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'time_from_value' => (new \DateTimeImmutable('2025-12-02T15:00:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'time_to_value' => (new \DateTimeImmutable('2025-12-02T16:30:00+0100'))->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'spots' => 10,
+      ],
+    ]);
+    $node->save();
+
+    $this->createOrders($node,
+      [
+        'lines' => [
+          [
+            'dates_delta' => 0,
+          ],
+        ],
+      ],
+      [
+        'email' => 'test-customer@example.com',
+        'lines' => [
+          [
+            'dates_delta' => 1,
+            'quantity' => 7,
+          ],
+        ],
+      ],
+
+      // Sell out.
+      [
+        'lines' => [
+          [
+            'dates_delta' => 2,
+            'quantity' => 10,
+          ],
+        ],
+      ],
+    );
+  }
+
+  /**
+   * Create orders in pretix.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   The public meeting node.
+   * @param array ...$specs
+   *   Order specs, e.g.
+   *
+   *   <code>
+   *   [
+   *     [
+   *       // Optional email; defaults to 'test@example.com'.
+   *       'email' => 'test-customer@example.com',
```

Comment:
remove?

Reply:
I'm trying to add documentation here, and I intend to say “You can specify an email, but if you don't, `test@example.com` will be used as a default”. Maybe I failed …

Reply:
ah the @param stuff confused me